# **Welcome to My Github üñ•Ô∏è!!**

## üåê I'm Nour, a junior bioinformatician with a bachelor's degree in biochemistry.

### ‚≠ê About Me
I have a year worth of experience in genomics & transcriptomics with interest in personalized medicine and decoding the intersectionality between environment, demographic factors and disease, disorder & infection prevalence. I am proficient in designing pipelines for preprocessing and analyzing large-scale datasets, employing advanced tools for complex data analysis. My research interest lies in studying digestive diseases inclusively.

### ‚≠ê Technical Skills
* **Programming Languages:** R, Bash & Python
* **Omics Technologies:** WGS Analysis with Bash, Bulk RNA-Seq with R & Bash, Resistome Analysis with Bash, Variant Calling with Bash
* **Statistics Software:** SPSS, GraphPad Prism
* **Database:** SQL
* **Bioinformatics Tools:** SRA-toolKit, BWA, samtools, IGV, FreeBayes, viralrecon, DESeq2, TCGABiolinks, edgeR, rgitools, ggplot2, limma, samtools, TrimGalore

### ‚≠ê Projects
### 1. [Uncovering Therapeutic Targets and Key Biomarkers of Cancer from Transcriptome](https://github.com/nournahtay/rnaseq-biomarker-discovery)
Bulk RNA-Seq Analysis (Differential Expression & Functional Enrichment Analyses) on TCGA datasets (glioblastoma, melanoma & glioma) to uncover biomarkers for therapeutic targetting. This work delves into the impact of demography (particularly sex) on the variation of early & late-stage biomarkers (in this case melanoma) and also verifies the status of isocitrate dehydrogenase on the progression of diffuse glioma. 

### 2. [Assembling & Variant Calling Whole SARS-CoV-2 Genomes](https://github.com/nournahtay/wgs-variant-calling)
An end-to-end analysis pipeline for assembling, annotating and variant-calling done on SARS-CoV-2 WGS data to identify & characterize mutations and their locations.


### Thank you for reaching this far ‚ú® 
